base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ScriptGeneratorViewModel.java

```diff
@@ -767,25 +746,6 @@ protected String getFirstNLinesOfInvalidityErrors(int i) {
      * @param viewTable The table in the view to update.
      */
     protected void updateValidityChecks(ActionsViewTable viewTable) {
-    List<ScriptGeneratorAction> actions = scriptGeneratorModel.getActions();
```

Comment:
Moved into the update calls in the various columns as this seemed more in-keeping with the swt tables